components/admin_console/plugin_management/plugin_management.tsx

```diff
@@ -413,8 +459,7 @@ export default class PluginManagement extends AdminSettings {
             resolveRemoveModal: null,
         });
     }
-
-    getConfigFromState = (config) => {
+    getConfigFromState = (config: Props['config']) => {
```

Comment:
Same here for the `Config` type

Reply:
@devinbinnie  should i refer to this file `mattermost-redux/types/config.d.ts`? I found that the keys that are required for this file are not available in the *Config* type

Reply:
Yes that's the correct type, which keys are missing? We can add those fields to the redux type.

Reply:
**PluginSettings**, **ExperimentalSettings** are the missing keys

`config: {
        PluginSettings: {
            Enable: boolean;
            EnableUploads: boolean;
            AllowInsecureDownloadUrl: boolean;
            EnableMarketplace: boolean;
            EnableRemoteMarketplace: boolean;
            AutomaticPrepackagedPlugins: boolean;
            MarketplaceUrl: string;
            RequirePluginSignature: boolean;
        };
        ExperimentalSettings: {
            RestrictSystemAdmin: boolean;
        };
    };
`

Reply:
@sowmiyamuthuraman thanks for handling. Have you had a chance to consider my comment here?
@nevyangelova some of the keys are missing in the config type. Waiting for reply from @devinbinnie

Reply:
Hey sorry for the late reply @sowmiyamuthuraman, I've been preoccupied.

Just spent some time digging through the `admin_settings` code, and for some reason they're using a different type definition for the `Config` object than `mattermost-redux`

This feels like something we'll need to cleanup separately, so we can make use of the types you've already defined for this conversion. Thanks for all your work on this :)